config/environment.js

```diff
@@ -11,7 +11,7 @@ module.exports = function(environment) {
       'script-src': "'self' https://*.firebaseio.com",
       'style-src': "'self' http://fonts.googleapis.com"
     },
-    firebase: 'https://embernati-lightning.firebaseio.com/',
+    firebase: 'https://dev-ember-lightning.firebaseio.com/',
```

Comment:
Is this just a sample one you created on your account?  That should work fine if so, was just curious.